src/components/missions.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Mission from './common/mission';
+import { getMissions, getMissionsFromApi } from '../redux/mission/mission';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+
+
+const Missions = ({ store }) => {
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if(store.length<0){
```

Comment:
- [ ]  I think you need to change this condition. because with this condition you put we will never get data from the API. you may even say `if(store.length !== 0)`